### PR TITLE
Move mentions of pre-standard revisions to an appendix.

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

**Review by akeranen — APPROVED** [review]

File: sdf.md [hunk 1]
```diff
@@ -2188,6 +2181,34 @@ using the `sdfThing` quality.
...
+
+* The quality `unit` was previously called `units`.
+* `sdfType` was developed out of a concept previously called `subtype`.
+* `sdfChoice` is the preferred way to represent JSO `enum` (only a
+  limited form of which is retained), and also the way to represent
+  JSO `anyOf`.
+* the length of text strings (as used with `minLength`/`maxLength`
+  constraints) was previously defined in bytes.
+  It now is defined as the number of characters (Unicode scalar
+  values, to be exact): a length in bytes is not meaningful unless
```
Comment (JKRhb) [inline sdf.md:2208]: Is a colon the right choice here, or should it rather be a period/semicolon?

**Review by JKRhb — APPROVED** [review]
See only one very minor point below.

File: sdf.md (hunk 1 above)
Comment (cabo) [reply sdf.md:2208]: Both works, but semicolon probably is more conventional. Can you do a suggestion in your comment?  I can't really edit right now.

Comment (JKRhb) [reply sdf.md:2208]: Sure, here you go :)

```suggestion
  values, to be exact); a length in bytes is not meaningful unless
```

**Pull request merged**